src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -199,10 +201,7 @@ class RideConstructionWindow final : public Window
 {
 private:
     uint8_t _currentlyShowingBrakeOrBoosterSpeed{};
-    uint64_t _currentDisabledSpecialTrackPieces{};
-    track_type_t _currentPossibleRideConfigurations[64]{};
-    uint16_t _numCurrentPossibleRideConfigurations{};
-    uint16_t _numCurrentPossibleSpecialTrackPieces{};
+    SpecialElementsList SpecialElementList;
```

Comment:
this should still have a _ prefix as its a private member variable. `SpecialElementsList _specialElementsList`. List implies linked list I would avoid that term for this. Perhaps work in Array or tbh its probably pretty obvious if its plural `_specialElements`.

Reply:
I will call it SpecialElementsDropdownState